etc/generate-notices.pl

```diff
@@ -1,438 +0,0 @@
-#!/usr/bin/env perl
```

Comment:
RIP to the last perl remaining in the repo.

Reply:
:moment-of-silence: